src/main/scala/org/broadinstitute/dsde/firecloud/service/OAuthService.scala

```diff
@@ -93,7 +93,7 @@ trait OAuthService extends HttpService with PerRequestCreator with FireCloudDire
 
             // we can't use the standard externalHttpPerRequest here, because the requestContext doesn't have the
             // access token yet; we have to add the token manually
-            val pipeline = addCredentials(OAuth2BearerToken(accessToken)) ~> sendReceive
+            val pipeline = addFireCloudCredentials ~> addCredentials(OAuth2BearerToken(accessToken)) ~> sendReceive
```

Comment:
I believe this is only used to talk to rawls, so the FC credentials are not needed here.